doc/with.md

```diff
@@ -5,33 +5,34 @@
 
 @signature `value.with( initialValue )`
 
-Creates an observable value that can be read, written, and observed using [can-reflect].
+  Creates an observable value that can be read, written, and observed using [can-reflect].
```

Comment:
I don't think we really should show using `canReflect` unless we need to ...  folks should use `.value` to read and write.